torch/fx/experimental/symbolic_shapes.py

```diff
@@ -277,13 +277,21 @@ def find_symbol_binding_fx_nodes(graph):
         if is_symbol_binding_fx_node(node)
     }
 
-def definitely_true(a):
+def definitely_true(a, *, never_guard=False):
```

Comment:
Alt: give this its own util

Reply:
...or at least remove the double negation :D

Reply:
There is a similar function in `_inductor /sizevars.py` called `is_expr_static_and_true` and it also does `_maybe_evaluate_static` which seems better.